### DISCUSSION TITLE
Query about features in the free tier

### DISCUSSION THREAD

**Discussion by mahithare** [community/community#88600] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a github organization which includes 16 private repositories. I need to add few company members into the organization. Can i add 10 users for free? I read somewhere that user limit is only upto 3 for read/write/admin permissions in the free tier but i have also seen unlimited users for private repositories. what is this contradiction? can i add more users who can just create repos and clone the existing repos? And also about storage, is there any limitation on total amount of storage of all the private repositories in an organization in the free tier?

**Comment by Shahnawazk7944** [comment]
You can add as many members as you want to your organization on the free tier but only 3 members can have full read/write/admin permissions on private repositories and others will have limited permissions (outside collaborators) like creating repos or cloning existing ones and the free tier provides a total of 500 MB storage for all private repositories within your organization.

**Reply by mahithare** [reply]
Thank you for the response! Where can i find the size of a repo in github? I am only the member of my org, not an admin. I downloaded one of the repositories, that itself is 1.1Gb and there are 15 more repos in the organization. So what is with the 500Mb?

**Reply by Shahnawazk7944** [reply]
It's not possible, The 500 MB limit applies to private repositories in the free tier for organizations. Public repositories have no storage limit and If you downloaded a 1.1 GB repository, it's likely public or your organization is on a paid plan with higher storage capacity

**Reply by Shahnawazk7944** [reply]
Use the following command to find out the size of the repository `git count-objects -vH`

**Reply by mahithare** [reply]
It says 500Mb of packages storage but not the size of a repository right? is there any storage limitation for a repository?

**Reply by Shahnawazk7944** [reply]
You're absolutely right! The 500 MB limit you encountered refers to package storage, not individual repository size and there's no direct storage limit for a single repository on GitHub, regardless of whether it's private or public.

**Reply by Shahnawazk7944** [reply]
Keep in mind that there is a 500 MB package storage limit for both public and private packages, as well as a total limit of 500 MB for private repositories within the organization.

**Reply by mahithare** [reply]
i don't see any 500mb limit on the private repositories. There is a limit of 50mb for each file, not the repo.
